### ISSUE TITLE
What is exploding gradient and how to prevent 

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#192]
If the weights in a network are very large, then the gradients for the lower layers involve products of many large terms. In this case you can have exploding gradients: gradients that get too large to converge.

Batch normalization can help prevent exploding gradients, as can lowering the learning rate.
